internal/loadbalancers/infrastructure/manager.go

```diff
@@ -0,0 +1,166 @@
+package infrastructure
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+
+	lbaas "github.com/equinix/metal-cli/internal/loadbalancers/api/v1"
+)
+
+const ProviderID = "loadpvd-gOB_-byp5ebFo7A3LHv2B"
+
+var LBMetros = map[string]string{
+	"da": "lctnloc--uxs0GLeAELHKV8GxO_AI",
```

Comment:
Let's make sure to get DC added in here `lctnloc-1lJjVT6Zp_Fs4UpW5LWQu` while we sort out how to not manage this manually.

Reply:
Assuming that the LB client gets moved over to `equinix-sdk-go`, the `ProviderID` constants and `LBMetros` var should be moved to the LB client (but as mentioned in a different comment, the Manager type should not be part of the shared client b/c CLI users will likely want more granular operations than the ones we implemented for CPEM).

Reply:
Seems like we decided to go with the internal clients for now.
Also, dc is in here now.